css/style.css

```diff
@@ -0,0 +1,129 @@
+.main,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+#projects {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+}
+
+.project-tile {
+  align-content: center;
+  display: flex;
+  flex-direction: row;
+
+  /* border: 1px solid black; */
+  margin-left: 7%;
+}
+
+nav div {
+  margin-left: 70%;
+}
+
+.project-tile div {
+  margin-left: 30px;
+  border: 1px solid black;
+}
+
+img {
+  width: 400px;
+  height: 420px;
+  margin: 0;
+  padding: 0;
+}
+
+h3 {
+  background-color: black;
+  color: white;
+  text-align: center;
+  margin: 0;
+  padding: 0;
+  padding: 15px;
+  border: 1px solid red;
+}
+
+nav {
+  background-color: red;
+  padding: 20px 0 20px 0;
+  margin: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+.nav-link {
+  color: white;
+  padding: 10px;
+  margin-left: 10px;
+  text-decoration: none;
+  font-size: 20px;
+  font-weight: 500;
+}
+
+.nav-link:hover {
+  background-color: rgb(49, 15, 15);
+}
+
+#welcome-section {
+  height: 100vh;
+  background-color: rgb(46, 10, 10);
+  text-align: right;
+  margin: 0;
+}
+
+h1 {
+  text-align: center;
+  color: white;
+  font-size: 40px;
+  padding-top: 15%;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+#wedd {
+  text-align: center;
+  color: white;
+  font-style: italic;
+  font-size: 25px;
+  margin-top: 0;
+}
+
+@media (max-width: 600px) {
+  body {
+    background-color: lightblue;
+  }
+
+  .project-tile {
+    align-content: center;
+    display: flex;
+    flex-direction: column;
+
+    /* border: 1px solid black; */
+    margin-left: 7%;
+  }
+}
+
+/* --------------- */
+
+/* img {
+  width: 400px;
+  height: 420px;
+  margin: 0;
+  padding: 0;
+} */
+
+/* img {
+  width: 400px;
+  height: 420px;
+  margin: 0;
+  padding: 0;
+} */
+
+/* .project-tile {
+  align-content: center;
+  display: flex;
+  flex-direction: row; */
```

Comment:
- [ ] The CSS file has lot of comment line , make sure to remove them to have a clean code , check them from line 110-129